ingest/src/test/java/com/microsoft/azure/kusto/ingest/IngestClientImplTest.java

```diff
@@ -59,16 +58,10 @@ void tearDown() {
     @Test
     void ingestFromBlob() {
```

Comment:
test name should indicate what are you testing. Each method have several tests, for happy flows, errors, etc.
You should follow java unit test naming convention.

Reply:
We have another branch to improve tests, including naming.